src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red, Yellow, Green, Blue, White
```

Comment:
enum values must be named using constants naming conventions:
Red -> RED, Blue -> BLUE, etc.